### PR TITLE
Font swapping

### PR DESCRIPTION
**Pull request by blakmarkit**
added font swapping script based on script by Shane Murphy @ prender.co and updated Extendscript API documentation URL in all scripts

### CODE REVIEW

**Review by ariestav — CHANGES_REQUESTED** [review]
No need to include the .vscode/launch.json or the ExtendScript/launch.json files in the repo.  Can you untrack them and add to .gitignore so git does not pick them up any longer?

**Comment by blakmarkit** [comment]
branch should be updated.

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,8 @@
 .DS_Store
 node_modules
 *.monopic
+/.vscode/launch.json
+/Extendscript/launch.json
```
Comment (ariestav) [inline .gitignore]: This needs "Extendscript" to be "ExtendScript" I think

Comment (blakmarkit) [reply .gitignore]: you are correct. updated.

**Pull request merged**